packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.js

```diff
@@ -68,6 +71,7 @@ export class SetupFlopFlipProvider extends React.PureComponent {
             : ldClientSideIdStaging,
           this.props.user && this.props.user.id,
           this.props.projectKey,
+          this.props.flags,
```

Comment:
Note, that `flags` are part of `adapterArgs` in `flopflip` as not all adapter do, will and can support this feature meainingfully.